performance/font-loading.md

```diff
@@ -0,0 +1,56 @@
+# Font Loading
+
+## Are you sure you need to use webfonts?
+
+It's worth asking this question initially. What benefit do you believe a webfont will give you and is it worth the cost in page performance? Afterall, system fonts are battle tested and were produced with legibility in mind.
```

Comment:
I love the fact that this is the first item. I would suggest rephrasing it so the title is not a question, to something like:


> ## When to use webfonts
> 
> Before using webfonts it's worth considering what benefits webfonts provide, and what impact they may have on things like accessibility or page speed. For example:
> 
> * System fonts are battle tested and were produced with legibility in mind. Using webfonts may reduce the readability of the site, especially for people with dyslexia.
> * The negative impact of webfonts (additional requests and increased page weight) on page load times may cause a page to go from "very slow" to "unusable", especially in markets of strategic importance for Springer Nature like China. This will in turn impact usage numbers.

Reply:
Thanks, great suggestion. I have reworded it a bit more as well to try and account for the fact that a person or team might have their own specific reasons for why the webfont might benefit their project, and also to communicate that it is advised to weigh up the pros and cons against each to ensure on balance the pros win.